utils/check_inits.py

```diff
@@ -12,13 +12,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
+Utility that checks the custom inits of Transformers are well-defined: the objects defined in both halves are the same.
```

Comment:
It seems we never define what `custom inits` means explicitly. Might be a good chance to state it here, so the docstring below for `parse_init` is super clear.

(Hope it's not my poor knowledge leading to my confusion)